src/QirRuntime/samples/StandaloneInputReference/qir-driver.cpp

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include <iostream>
+#include "CLI11.hpp"
+#include "QuantumApi_I.hpp"
+#include "SimFactory.hpp"
+#include "QirContext.hpp"
+
+using namespace Microsoft::Quantum;
+
+extern "C" void Quantum__StandaloneSupportedInputs__ExerciseInputs__body( // NOLINT
+    int64_t anInt,
+    double aDouble);
+
+int main(int argc, char *argv[])
+{
+    try{
+        CLI::App app("QIR Standalone Entry Point Inputs Reference");
+
+        // Add the --simulator-output and --operation-output options.
+        // N.B. These options should be present in all standalone drivers.
+        std::string simulatorOutputFile;
+        CLI::Option *simulatorOutputFileOpt = app.add_option(
+            "-s,--simulator-output",
+            simulatorOutputFile,
+            "File to write the output of the simulator to");
+
+        std::string operationOutputFile;
+        CLI::Option *operationOutputFileOpt = app.add_option(
+            "-o,--operation-output",
+            operationOutputFile,
+            "File to write the output of the Q# operation to");
+
+        // Add the options that correspond to the parameters that the QIR entry-point needs.
+        // Option for a Q# Int type.
+        int64_t anInt = 0;
+        CLI::Option *anIntOpt = app.add_option(
+            "--anint",
```

Comment:
Just curious, why include article in the name of the argument? It doesn't make sense for non-English speakers and makes the names harder to parse...

Reply:
Fixed it.